ipam/space/space.go

```diff
@@ -122,8 +122,11 @@ func (s *Space) biggestFreeRange(r address.Range) (biggest address.Range) {
 	biggestSize := address.Count(0)
 	s.walkFree(r, func(chunk address.Range) bool {
 		if size := chunk.Size(); size >= biggestSize {
-			biggest = chunk
-			biggestSize = size
+			chunk = chunk.BiggestPow2AlignedRange()
```

Comment:
The first if-statement discards chunks that cannot possibly contain the biggest CIDR-aligned range.
The second if-statement accepts the biggest CIDR-aligned range.

Yes, it is coded this way to run faster than if we looked for the biggest CIDR-aligned chunk in each free chunk.